tests/operator/tests/operator_install_status.go

```diff
@@ -0,0 +1,205 @@
+package operator
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalhelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalparameters"
+	tshelper "github.com/test-network-function/cnfcert-tests-verification/tests/operator/helper"
+	tsparams "github.com/test-network-function/cnfcert-tests-verification/tests/operator/parameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/execute"
+)
+
+var _ = Describe("Operator install-source,", func() {
+
+	var (
+		installedLabeledOperators []tsparams.OperatorLabelInfo
+	)
+
+	execute.BeforeAll(func() {
```

Comment:
Based on testing these cases with the rest of the operator suite I think we should clean the operator namespace before deploying anything.
```suggestion
	execute.BeforeAll(func() {
	        By("Clean namespace")
		err := namespaces.Clean(tsparams.OperatorNamespace, globalhelper.GetAPIClient())
		Expect(err).ToNot(HaveOccurred())
```